src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -233,6 +278,43 @@
     text: az backup protection enable-for-azurefileshare --policy-name MyPolicy --resource-group MyResourceGroup --vault-name MyVault --storage-account MyStorageAccount --azure-file-share MyAzureFileShare
 """
 
+helps['backup protection enable-for-azurewl'] = """
+type: command
+short-summary: Start protecting a previously unprotected workload within an Azure VM as per the specified policy to a Recovery services vault. Provide the workload details as a protectable item.
+examples:
+  - name: Start protecting a previously unprotected workload within an Azure VM as per the specified policy to a Recovery services vault. Provide the workload details as a protectable item.
+    text: az backup protection enable-for-azurewl --policy-name MyPolicy --resource-group MyResourceGroup --vault-name MyVault --protectable-item-name ItemName --protectable-item-type SQLInstance --server-name Myserver --workload-type MSSQL
+"""
+
+helps['backup protection auto-enable-for-azurewl'] = """
+type: command
+short-summary: Automatically protect all existing unprotected DBs and any DB which will be added later with the given policy. Azure backup service will regularly scan auto-protected containers for any new DBs and automatically protect them.
```

Comment:
Actually your short summary is too long and it will be better if you can refine it.

Reply:
@sambitratha : "Automatically protect all existing unprotected DBs and any future DBs with the given policy"